lighthouse-core/lib/cdt/generated/SourceMap.js

```diff
@@ -32,100 +32,32 @@ const Common = require('../Common.js')
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 Object.defineProperty(exports, "__esModule", { value: true });
-
-/**
- * @interface
- */
-class SourceMap {
-    /**
-     * @return {string}
-     */
-    compiledURL() {
-    }
-    /**
-     * @return {string}
-     */
-    url() {
-    }
-    /**
-     * @return {!Array<string>}
-     */
-    sourceURLs() {
-    }
-    /**
-     * @param {string} sourceURL
-     * @return {?string}
-     */
-    embeddedContentByURL(sourceURL) {
-    }
-    /**
-     * @param {number} lineNumber in compiled resource
-     * @param {number} columnNumber in compiled resource
-     * @return {?SourceMapEntry}
-     */
-    findEntry(lineNumber, columnNumber) {
-    }
-    /**
-     * @param {string} sourceURL
-     * @param {number} lineNumber
-     * @param {number} columnNumber
-     * @return {?SourceMapEntry}
-     */
-    sourceLineMapping(sourceURL, lineNumber, columnNumber) {
-    }
-    /**
-     * @return {!Array<!SourceMapEntry>}
-     */
-    mappings() {
-    }
-    dispose() {
-    }
-}
-
-/**
- * @unrestricted
- */
+exports.TextSourceMap = exports.SourceMapEntry = exports.Offset = exports.Section = exports.SourceMapV3 = void 0;
+;
+;
+;
+;
+;
+;
+;
+;
+;
 class SourceMapV3 {
     constructor() {
-        /** @type {number} */ this.version;
-        /** @type {string|undefined} */ this.file;
-        /** @type {!Array.<string>} */ this.sources;
-        /** @type {!Array.<!SourceMapV3.Section>|undefined} */ this.sections;
-        /** @type {string} */ this.mappings;
-        /** @type {string|undefined} */ this.sourceRoot;
-        /** @type {!Array.<string>|undefined} */ this.names;
     }
 }
-/**
- * @unrestricted
- */
-SourceMapV3.Section = class {
+exports.SourceMapV3 = SourceMapV3;
+class Section {
     constructor() {
-        /** @type {!SourceMapV3} */ this.map;
-        /** @type {!SourceMapV3.Offset} */ this.offset;
     }
-};
-/**
- * @unrestricted
- */
-SourceMapV3.Offset = class {
+}
+exports.Section = Section;
+class Offset {
     constructor() {
-        /** @type {number} */ this.line;
-        /** @type {number} */ this.column;
     }
-};
-/**
- * @unrestricted
- */
+}
+exports.Offset = Offset;
 class SourceMapEntry {
-    /**
-     * @param {number} lineNumber
-     * @param {number} columnNumber
-     * @param {string=} sourceURL
-     * @param {number=} sourceLineNumber
-     * @param {number=} sourceColumnNumber
-     * @param {string=} name
-     */
     constructor(lineNumber, columnNumber, sourceURL, sourceLineNumber, sourceColumnNumber, name) {
```

Comment:
one downside of the switch to compiled typescript is we've lost all types for source maps :/